src/cpp/session/SessionSuspend.cpp

```diff
@@ -123,6 +127,17 @@ void resetSuspendTimeout()
    s_suspendTimeoutTime = timeoutTimeFromNow();
 }
 
+void resetSuspendTimeout(boost::shared_ptr<HttpConnection> pConnection)
+{
+   if (s_suspendFilters.shouldResetSuspendTimer(pConnection))
+   {
+      LOG_DEBUG_MESSAGE(pConnection->request().uri() +
```

Comment:
I think this might be a little low on the "signal to noise" ratio since it will apply to all but a couple of URIs. We already will see in the logs before/after messages for each URI so I'd say this is probably not necessary.

Reply:
I would flip it here, if the suspend filters prevent the request from resetting the suspend timeout, log that. Seems like it would be the rarer of the two events.